### DISCUSSION TITLE
搜索界面屏幕旋转会闪退

### DISCUSSION THREAD

**Discussion by llwy142086** [WatsonYao/nian_app#120] [Q&A] [answered]
![mmexport1732968324845](https://github.com/user-attachments/assets/3ce1267f-5d21-489d-8282-72297ec1d1ec)
设备是三星平板tabs8，当时停留在进展搜索界面，无意识地把平板立起来后，发现app闪退了。

**Comment by WatsonYao** [comment, marked as answer]
这边排查看看，如果找到问题会在后续版本中修复。
